frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -149,23 +116,27 @@ export default {
       invisible_labels: [],
       // Command
       instance_list: undefined,
-      command_manager: undefined,
-      history: undefined,
     }
   },
-  computed: {
-    undo_disabled: function () {
-      return !this.history || !this.history.undo_posible
+  watch: {
+    instance_list: function (newVal) {
+      if (this.working_file.type === "audio" && newVal) {
```

Comment:
If `intance_list` changes save it in the `instance_store` so it can later be accessed by `annotation_ui_facotry` for saving functionality. This logic was copied from `text_annotation_core`